docs/2.build/4.web3-apps/chain-ui.md

```diff
@@ -0,0 +1,77 @@
+---
+id: chain-deployed-ui
+title: Chain Deployed UI
+sidebar_label: Chain Deployed UI
+---
+
+[Chain Deployed UI](https://github.com/near/chain-deployed-ui) is a [decentralized frontend solution](frontend.md#decentralized-frontend-solutions) that lets you to deploy dApp frontends directly on the NEAR blockchain. A frontend can be built with typical Web2 tooling (e.g. React + Vite + node.js dependencies), and the resulting compressed bundle can be stored in the state of a smart contract.
```

Comment:
Would this be a good place to call out that this is pre-alpha/experimental? 😅

Reply:
From our recent chat, we want to clarify with @mpeterdev about how we want to communicate the "Chain deployed UI" readiness.

Since it's in a pre-alpha /experimental stage, I think we should keep this documentation as a draft, and when we think it's “ready to go” then we can publish the docs.

I'm adding a note to this article, even if we keep it as a draft.